### ISSUE TITLE
"Defining ZeroLength element"

### ISSUE THREAD

**Issue by sadeghebrahimi** [rclab-auth/gidopensees#57]
Dear GiD+OpenSees team;

I have the same problem as the issue number 25, but I am not sure that closed issues are still available for new comments, so I made another one.  

I was faced with this error: "ZeroLength Element with only 1 node. ZeroLength elements must be assigned to at least 2 nodes." when I tried to create zerolength elements. 
I have 2 questions.
1- How can define 2 points at the same location?
2- How should assign ZeroLength element to those 2 points? Assigning 2 zerolength element to those 2 points with the same ID?

Thank you for your consideration.

**Comment by vpapanik** [comment]
1. It is not necessary for two points to share the exact same location for the ZL element to work. However you can still create two distinct points in the same coordinates with GiD, by selecting 'no join' when you create the second point upon the first one. If you enable point labels you will see two IDs one upon another. Another trick to handle double points is to assign each point to a different layer so you can easily turn on and off the visibility of each point.

2. You can just select those two points and assign them to the one ZL element.

First try with two distinct points (not same coordinates) to get the general idea.

**Comment by sadeghebrahimi** [comment]
Dear Prof. Papanikolaou

I really appreciate your helpful explanation.

I applied different layers to each point which leads to define ZL elements easily.
Then, I created .tcl file.
In the file, the command of ZL element is: 
element zeroLength $eleTag $iNode $jNode -mat $matTag1 $matTag2 ... -dir $dir1 $dir2 ...

My question is about the full command of ZL element.
element zeroLength $eleTag $iNode $jNode -mat $matTag1 $matTag2 ... -dir $dir1 $dir2 ...<-doRayleigh $rFlag> <-orient $x1 $x2 $x3 $yp1 $yp2 $yp3

Is there any way to define "vector components in global coordinates defining local x-axis" , i.e. x1, x2, and x3 in this interface? I have the same question for "vector components in global coordinates defining vector yp which lies in the local x-y plane for the element" (y1, y2, and y3).
In other words, does the GiD+OpenSees interface have the ability of determination of local x-axis and y-axis of ZL elements?

Thank you for your consideration.

**Comment by vpapanik** [comment]
No, unfortunately this is not covered yet. However, you can assign ZL properties for all six different degrees of freedom in global axes.
